common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -93,13 +93,13 @@ function (Iterator) {
          * initial configuration.
          * @param {array} speeds List of speeds available for the player.
          */
-        render: function (speeds) {
+        render: function (speeds, speed) {
```

Comment:
@muzaffaryousaf I've added a doc string and updated the variable name. We already have tests that check for the functionality. The test checks for the appropriate class, but it doesn't check the styles. I think this is OK, as long as the correct markup is present.

I'm updating the sandbox right now.